app/state/selectors/authSelectors.js

```diff
@@ -40,16 +47,13 @@ const staffUnitsSelector = createSelector(
 );
 
 /**
- * Gotcha warning: Respa API uses isStaff boolean value for admin users.
- *
- * From Varaamo's point of view, a user is staff if they have permissions to
- * approve reservations for a resource.
+ * Check if a user has admin permission for a unit.
+ * TODO: Find a better name for this.
  */
 function createIsStaffSelector(resourceSelector) {
   return createSelector(
     resourceSelector,
-    staffUnitsSelector,
-    (resource, staffUnits) => includes(staffUnits, resource.unit)
+    resource => Boolean(resource && resource.userPermissions && resource.userPermissions.isAdmin)
```

Comment:
```suggestion
    resource => Boolean(resource && resource.userPermissions && resource.userPermissions.isAdmin)
```
`get(resource, 'userPermissions.isAdmin', false)`